rest/replicatortest/replicator_test.go

```diff
@@ -8681,6 +8681,9 @@ func TestReplicationConfigUpdatedAt(t *testing.T) {
 	currTime := configResponse.UpdatedAt
 	createdAtTime := configResponse.CreatedAt
 
+	// avoid flake where update at seems to be the same (possibly running to fast)
+	time.Sleep(500 * time.Nanosecond)
```

Comment:
We should bump this more - we can tolerate a few extra milliseconds of test time no problem

```suggestion
	// avoid flake where update at seems to be the same (possibly running to fast)
	time.Sleep(10 * time.Millisecond)
```